### PR TITLE
Changes and improvements to the documentation

### PR DESCRIPTION
**Pull request by Blatko1**
I have made changes to the documentation to increase clarity and enhance the reading experience. Some of the changes are: fixed typos, removed/added commas where needed, changed/added articles before words where needed, broke down some sentences.

> -  Tested on all platforms changed - **not needed**
> - Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users - **not needed**
> - Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior - **not needed**
> - Created or updated an example program if it would help users understand this functionality - **not needed**
> - Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented - **not needed**

### CODE REVIEW

**Review by madsmtm — APPROVED** [review]
Not a native speaker, but this looks fine to me

File: src/lib.rs [hunk 1]
```diff
@@ -183,13 +183,13 @@ pub mod window;
...
+/// and `Sync`, since they always execute on a single thread.
 ///
 /// # Safety
 ///
-/// Winit can run only one event loop at the time and the event loop itself is tied to some thread.
-/// The objects could be send across the threads, but once passed to winit, they execute on the
-/// mean thread if the platform demands it. Thus marking such objects as `Send + Sync` is safe.
+/// Winit can run only one event loop at a time, and the event loop itself is tied to some thread.
+/// The objects could be sent across the threads, but once passed to winit, they execute on the
+/// mean thread if the platform demands it. Thus, marking such objects as `Send + Sync` is safe.
```
Comment (daxpedda) [inline src/lib.rs:192]:
```suggestion
/// main thread if the platform demands it. Thus, marking such objects as `Send + Sync` is safe.
```

**Review by daxpedda — APPROVED** [review]
Also not a native speaker, but LGTM.
Thank you for doing this!

**Comment by madsmtm** [comment]
Thanks!

**Pull request merged**